storage_service/locations/models/package.py

```diff
@@ -589,6 +722,12 @@ def backlog_transfer(self, origin_location, origin_path):
             source_path=os.path.join(self.origin_location.relative_path, self.origin_path),
             destination_path=self.current_path,  # This should include Location.path
             destination_space=dest_space)
+
+        try:
+            self.index_file_data_from_transfer_mets(prefix=os.path.join(dest_space.staging_path, self.current_path))  # create File entries for every file in the transfer
+        except StorageException as e:
+            LOGGER.warning("Transfer METS data could not be read: %s", str(e))
```

Comment:
Within that AM version, yep! After an upgrade to Archivematica 1.4, they'll need to be reindexed in the storage service using the script that's there.